site/content/docs/developing/addons/package.en.md

```diff
@@ -0,0 +1,140 @@
+# Uploading Local Artifacts <span class="version" > added in [v1.21.0](../../../blogs/release-v121.en.md) </span>
+
+Copilot supports uploading local files referenced from your addon templates to S3, and replacing the relevant resource properties with the uploaded S3 location.
+On [`copilot svc deploy`](../../commands/svc-deploy.en.md) or [`copilot svc package --upload-assets`](../../commands/svc-package.en.md), certain fields on supported resources will be updated with an S3 location.
+To see the full list of resources that are supported, take a look at the [AWS CLI documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/package.html).
```

Comment:
I read this as saying that my local manifests would be updated:

> certain fields on supported resources will be updated with an S3 location

But that's not the case... if this could be clarified that would be great!